src/vs/workbench/browser/parts/editor/tabFocus.ts

```diff
@@ -21,14 +21,18 @@ export class TabFocusMode extends Disposable {
 
 	constructor(
 		@IContextKeyService contextKeyService: IContextKeyService,
-		@IConfigurationService configurationService: IConfigurationService
+		@IConfigurationService private readonly _configurationService: IConfigurationService
```

Comment:
Don't think this change was needed?

Reply:
yep, needed because used within here https://github.com/microsoft/vscode/blob/65194b1e7c201a71a54fe2f203d116e99c7ff2d1/src/vs/workbench/browser/parts/editor/tabFocus.ts#L54-L63

Reply:
But that's still in the constructor, so it should be available without making it a member

Reply:
In fact it is used without this on line 56

Reply:
yeah you're right - weird because before when I tried this, it was saying that's not accessible within the scope

Reply:
probably because I tried `this.configurationService`